packages/dds/tree/src/test/shared-tree/sharedTreeFuzzTests.ts

```diff
@@ -227,8 +230,15 @@ function runBatch(
 	for (let i = 0; i < batchSize; i++) {
 		const runSeed = seed + i;
 		const generatorFactory = () => take(opsPerRun, opGenerator());
+		const saveInfo: SaveInfo = {
+			saveOnFailure: false, // Change to true to save failing runs.
+			saveOnSuccess: false, // Change to true to save successful runs.
+			filepath: resolve(
+				`./src/test/shared-tree/fuzz-tests-saved-ops/ops_with_seed_${runSeed}`,
```

Comment:
i'm not sure resolving with relative path to cwd is safe for all workflows/ways one might run the test command. i've generally preferred using a path relative to `__dirname` (which does get a bit ugly since the file being executed is inside the dist folder).